lib/helpers/remote-config.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+import '../global.dart';
+
+Future<dynamic> getApplicationConfiguration(String key) async {
+  final RemoteConfig remoteConfig = await RemoteConfig.instance;
```

Comment:
This shouldn't be inside the function, it appears to be adding an additional await on every call.

Reply:
created two classes with two different responsibilities to save this initialize on each call..